New package: MDSh5 v0.1.0

JuliaRegistrator opened the pull request:
- Registering package: MDSh5
- Repository: https://github.com/anchal-physics/MDSh5.jl
- Created by: @anchal-physics
- Version: v0.1.0
- Commit: 4cc79836d7ba7b0ed011a67d07ba458b54763e77
- Reviewed by: @anchal-physics
- Reference: https://github.com/anchal-physics/MDSh5.jl/commit/4cc79836d7ba7b0ed011a67d07ba458b54763e77#commitcomment-157176772
- Description: Julia version of mdsh5 python package that uses PythonCall.jl.
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd6085ad44c1357993ce2ad612a2293267f8a4bcf6a156b31f92e3f38c254b41cb9a0de02c044c1f341646fb701a9b6f842576d85de6a5d6de23cd2732d0e7885e0492d49f8a86f6151336326b3c840374d218ec38fc82e9d6b89288533d68c2c44964e85118bd4192a0c38d9d0d993443d36d1d753e0c7ed69b2336da75914a62d87881a3df271c41de1c76e30817ad52c9a4ac1f5a632dff4490c60f782a166aba8e5648e2772cae9e34da0e632be2792 -->

anchal-physics commented:
Changed name and got a new PR https://github.com/JuliaRegistries/General/pull/131014

This PR can be closed now.

goerz commented:
[noblock] I would recommend sticking to the name, since

> Julia version of `mdsh5` python package

anchal-physics commented:
Well, I would very much like that. What do I need to do?

goerz commented:
[noblock] Yes, that's right. Revert the commit and rename the repo itself back. You also need to fix the `compat` entry for Julia, of course. Then, retrigger the registration, and the registrator should automatically update this PR.

JuliaRegistrator committed:
New package: MDSh5 v0.1.0

UUID: d26121ab-931e-479e-b928-38e5e8d76109
Repo: https://github.com/anchal-physics/MDSh5.jl.git
Tree: db7f24b89097da066a1d01bb100ebc52b5c1ddfc

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
